python/mxnet/gluon/rnn/rnn_cell.py

```diff
@@ -333,7 +333,7 @@ class RNNCell(HybridRecurrentCell):
         Initializer for the bias vector.
     h2h_bias_initializer : str or Initializer, default 'zeros'
         Initializer for the bias vector.
-    prefix : str, default 'rnn_'
+    prefix : str, default ```'rnn_'```
```

Comment:
Usually we ``` for a block and for single variable we use `. Did that fail?

Reply:
discussed offline, @frankfliu showed that a single ` turns into a link.
http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder

@frankfliu will change this to ``